web/index.html

```diff
@@ -94,7 +93,8 @@
 
     </div>
 
-    <!-- entry point dart file -->
-    <script type="application/dart" src="./viewer.dart"></script>
+    <script type="application/dart">
+        export 'viewer.dart';
```

Comment:
Any reason to do the export like this as opposed to using the src attribute? They should be effectively the same thing so I am just curious.